properties/service_fabric_managed_netstandard.targets

```diff
@@ -3,7 +3,8 @@
   
   <!-- Target to build net standard.-->
```

Comment:
You have not packaged the new DLL in the nuget package. You need to update the nuproj files.

Reply:
Fixed intendation and nuget addition. Not sure if I need to keep all the target paths in StartupServicesUtility.nuget or keeping netstandard2.0 and net45 should be enough.